keyboards/pimoroni/keybow2040/keymaps/default/keymap.json

```diff
@@ -0,0 +1,23 @@
+{
+    "keyboard": "pimoroni/keybow2040",
+    "author": "fanf2",
+    "version": 1,
+    "keymap": "default",
+    "layout": "LAYOUT",
+    "layers": [
+	[
+            "RGB_TOG", "RGB_MOD", "KC_C",    "KC_D",
+            "KC_E",    "KC_F",    "KC_G",    "KC_H",
+            "KC_I",    "KC_J",    "KC_K",    "KC_L",
+            "KC_M",    "KC_N",    "KC_O",    "KC_P",
+	    "KC_Q"
+	],
+	[
+            "KC_F1",    "KC_F2",    "KC_F3",    "KC_F4",
+            "KC_F5",    "KC_F6",    "KC_F7",    "KC_F8",
+            "KC_F9",    "KC_F10",   "KC_F11",   "KC_F12",
+            "KC_F13",   "KC_F14",   "KC_F15",   "KC_F16",
+	    "KC_F17"
+	]
```

Comment:
The second layer is mostly useless, because you don't have any way to switch to that layer.

Reply:
Heh, yes :-)

I am not sure what the best default for a macro pad should be: function keys seem benign and not totally useless for plug-and-play beginners, but I guess the most versatile config would be a load of dynamic macro keys that VIA can edit.

When I was using it to extend a HHKB I had F keys in the top half and a navigation cluster in the bottom half. The Pimoroni CircuitPython firmware needs configuration with Python to do anything.

Reply:
I have settled on a defensible default layout in round 2